### ISSUE TITLE
Exposed jar now deployed in Bintray

### ISSUE THREAD

**Issue by sdeleuze** [crazyproger/borsch#1]
Hey, I just would like to inform you that Exposed jar is now deployed in https://bintray.com/kotlin/exposed/exposed/view#, so you can remove the JAR from Git repository and use it in your build.

**Comment by crazyproger** [comment]
Yeah, thanks! Will fix later. I've start using bintray right after your question on Exposed).
